src/main/paradox/features/secrets.md

```diff
@@ -0,0 +1,34 @@
+## Secrets
+
+A non-blocking secrets API is available for Kubernetes. The secrets must be declared in the `build.sbt` file, and then they can be accessed at runtime using the provided libraries. This feature will be made available for DC/OS at a later date.
+
+#### Project
+
+For example, given the following `build.sbt` setting:
+
+```sbt
+secrets += Secret("my-secret", "my-key")
+```
+
+A developer can access this setting at runtime:
+
+```scala
+import com.lightbend.rp.secrets.scaladsl.SecretReader
+
+val secret: Future[Option[ByteString]] =
+  SecretReader.get(name = "my-secret", key = "my-key")
+```
+
+#### Kubernetes
+
+An operator can declare this secret before generating the deployment resources:
+
+```bash
+kubectl create secret generic my-secret --from-file=my-key=./path-to-my-secret-file
+```
+
+<link rel="stylesheet" type="text/css" href="../css/custom.css">
+
+@@@ warning
+Integration with all deployment targets except for Minikube is **Incubating** at this point.
+@@@
```

Comment:
the warnings all right below the heading btw, otherwise it's like hidden away and not that visible

Reply:
The whole page renders like this:

<img width="1152" alt="lo4" src="https://user-images.githubusercontent.com/184683/43649749-f372c362-970b-11e8-92a2-461331f5bf36.png">

It's consistent with other pages, and it's clear enough, I think.